### PR TITLE
Closes #614 | Add Dataloader LEXiTRON

### LINKED ISSUE #614: Create dataset loader for LEXiTRON
Dataloader name: `lexitron/lexitron.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?lexitron

|     Dataset| lexitron |
|-------------|---|
| Description | Corpus-based dictionary of Thai and English languages. This dataset contains frequently-used words from trusted publications such as novels, academic documents and newspaper. The dataset link contains Thai-English and English-Thai lexicons. Thai-English vocabulary consists of vocabulary, type of word (part of speech), translation, synonym (synonym) and sample sentences with a list of Thai-> English words, 53,000 words and English vocabulary list -> Thai, 83,000 words. See more details at http://lexitron.nectec.or.th. |
| Subsets | version 2.0 |
| Languages | tha, eng |
| Tasks | Word-level Translation, Machine Translation |
| License | Custom NECTEC license |
| Homepage | https://opend-portal.nectec.or.th/dataset/lexitron-2-0 |
| HF URL | - |
| Paper URL | - |

### PR DESCRIPTION
**Pull request by muhammadravi251001**
**Title**: Add Dataloader LEXiTRON

**First line PR Message**: Closes https://github.com/SEACrowd/seacrowd-datahub/issues/614

**Note**: There are no citations available for this dataset.

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/{my_dataset}/{my_dataset}.py` (please use only lowercase and underscore for dataset folder naming, as mentioned in dataset issue) and its `__init__.py` within `{my_dataset}` folder.
- [x] Provide values for the  `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_LOCAL`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py` or `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py --subset_id {subset_name_without_source_or_seacrowd_suffix}`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

File: seacrowd/sea_datasets/lexitron/lexitron.py [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+  title     = {},
+  journal   = {},
+  volume    = {},
+  year      = {},
+  url       = {},
+  doi       = {},
+  biburl    = {},
+  bibsource = {}
+}
+"""
```
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/lexitron/lexitron.py:51]:
```suggestion
# There are no citations available for this dataset.
_CITATION = ""
```

**Review by ljvmiranda921 — COMMENTED** [review]
Ran the data loader and it worked! Can you rerun `make check_file` for formatting again?

**Comment by muhammadravi251001** [comment]
> Ran the data loader and it worked! Can you rerun `make check_file` for formatting again?

It's already done Sir on https://github.com/SEACrowd/seacrowd-datahub/pull/646/commits/a185ae4a6d315524a5c70f44dc1a2c95b0629349 commit.

**Review by ljvmiranda921 — APPROVED** [review]
Thanks for applying the changes! LGTM

**Comment by muhammadravi251001** [comment]
> Thanks for applying the changes! LGTM

Thanks for the approval, Sir!

**Review by dehanalkautsar — APPROVED** [review]
Hi! I also tried to run the dataloader and it ran pretty well. I think the code itself is quite clean. LGTM!

**Comment by muhammadravi251001** [comment]
> Hi! I also tried to run the dataloader and it ran pretty well. I think the code itself is quite clean. LGTM!

Thanks for the approval, Sir!

**Comment by ljvmiranda921** [comment]
Merging in a few :)

**Pull request merged**